MediaWikiTelegramBot.js

```diff
@@ -284,7 +293,38 @@ function showCurrentMwMessage(userID) {
 function advanceMwMessage(userID) {
     const user = getUser(userID);
 
+    // Make sure there is another message to translate
+    if( user.currentMwMessageIndex + 1 >= user.loadedMwMessages.length ) {
+        user.currentMwMessageIndex = 0;
+        user.loadedMwMessages = [];
+
+        // Prepare the fetch untranslated button
+        const inlineKeyboard = [inlineKeyboardButton(
+            i18n(getLanguageCode(userID), "tgbot-load-messages"),
+            callbackPrefixes.LOAD_UNTRANSLATED
+        )];
+
+        const tgMsgOptions = {
+            reply_markup: JSON.stringify({
+                inline_keyboard: inlineKeyboard
+            })
+        };
+
+        tgBot.sendMessage(
+            userID,
+            i18n(getLanguageCode(userID), "tgbot-no-untranslated-messages-left"),
+            tgMsgOptions
+        );
+
+        return;
+    }
+
+    // Show them next message
     user.currentMwMessageIndex++;
+    tgBot.sendMessage(
+        userID,
+        'The next message is: '
```

Comment:
This must be in double quotes. This can also be enforced using eslint.

Besides, since you already know how to add messages to en.json and qqq.json, if would be very nice to add this one :)